test/unit-tests/function/matrix/map.test.js

```diff
@@ -16,6 +16,92 @@ describe('map', function () {
     assert.ok(Array.isArray(arr2))
   })
 
+  it('should map two arrays', function () {
```

Comment:
Thanks for the extensive testing!

I think we also need some test for the map.transform.js version of the function, or am I overlooking them?

Reply:
They are included in map.test.js the ones that state they should work from the parser from line 249 to 288. Is that ok?